MediumProject/src/main/java/org/example/stage2/Searching.java

```diff
@@ -0,0 +1,68 @@
+package org.example.stage2;
+
+import java.util.*;
+
+public class Searching {
+
+    private static List<String[]> listOfData(){
+        System.out.println("Enter the number of people:");
+        Scanner scan = new Scanner(System.in);
+        int size = scan.nextInt();
```

Comment:
if input will be -3?

Reply:
in the task it's impossible 
they don't have such case